project/Settings.scala

```diff
@@ -0,0 +1,39 @@
+import sbt._
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+/**
+ * Application settings. Configure the build for your application here.
+ * You normally don't have to touch the actual build definition after this.
+ */
+object Settings {
+  /** The name of your application */
+  val name = "anduin-fsm"
+
+  /** The version of your application */
+  val version = "1.0"
```

Comment:
The best practice is to define `version in ThisBuild := "1.0"` in a separate file (or use `sbt-git` to have it set automatically from git commit hash). The pros are:

* You could bump the version without touching other unrelated files.
* Version is applied to all defined projects in this build (including sub-projects).

Reference: [https://github.com/typelevel/cats/blob/master/version.sbt](https://github.com/typelevel/cats/blob/master/version.sbt) and [https://github.com/circe/circe/blob/master/version.sbt](https://github.com/circe/circe/blob/master/version.sbt).